src/Term/TermFallback.php

```diff
@@ -46,6 +46,18 @@ public function __construct( $requestedLanguageCode, $text, $actualLanguageCode,
 			throw new InvalidArgumentException( '$sourceLanguageCode must be a non-empty string or null' );
 		}
 
+		if ( $actualLanguageCode === $sourceLanguageCode ) {
```

Comment:
Do you understand that I think this could be a problem? Why should knowledge about the API be baked into the DataModel? @JeroenDeDauw recently blocked me for exactly that reason in #334.